src/components/ui/SuggestItem.jsx

```diff
@@ -10,23 +11,29 @@ const ItemLabels = ({ firstLabel, secondLabel }) =>
     <div className="autocomplete_suggestion__second_line">{secondLabel}</div>
   </div>;
 
-const GeolocationItem = () =>
+const GeolocationItem = ({ isHighlighted }) =>
```

Comment:
To avoid managing highlighted state on each component, you could have them render only the specific content of each item, and put in the parent the rendering of the standard list item container as a `<li>` with the `autocomplete_suggestion` and optional `selected` list.

Reply:
Sorry I should have probably done that in the PR where I separated data and rendering.

Reply:
I agree but this would lead us to a css refactoring, I prefer to let the style untouched in this PR